src/index.ts

```diff
@@ -0,0 +1,3 @@
+export type { default as Config } from "./Config";
+export * from "./engine";
+export type { default as Rule } from "./Rule";
```

Comment:
Should we export the other non-default types such as `SchemaConfig` or `Reporter` from the repository root?

Reply:
I guess. I don't think anyone is creating custom reporters at this point, but if they want to then it would be useful. I don't think this is urgent though.